compiler/rustc_lint/src/cast_ref_to_mut.rs

```diff
@@ -0,0 +1,58 @@
+use rustc_ast::Mutability;
+use rustc_hir::{Expr, ExprKind, MutTy, TyKind, UnOp};
+use rustc_middle::ty;
+
+use crate::{lints::CastRefToMutDiag, LateContext, LateLintPass, LintContext};
+
+declare_lint! {
+    /// The `cast_ref_to_mut` lint checks for casts of `&T` to `&mut T`
+    /// without inferior mutability.
```

Comment:
Is "inferior" a typo here, or some term I don't understand in this context?

Reply:
Just a typo. ;-)